src/bridge/MemberListSyncer.ts

```diff
@@ -421,17 +422,20 @@ export class MemberListSyncer {
                     return !ircUserIds.includes(userId);
                 }
             );
-
-            this.usersToLeave += usersToLeave.length;
+            if (usersToLeave.length < 1) {
```

Comment:
which is also this?
```suggestion
            if (usersToLeave.length === 0) {
```

Reply:
I like to think `< 1` is for those who are extra paranoid. But point.

Reply:
I normally do`<= 0` which people hate :p